baseplate/frameworks/queue_consumer/kombu.py

```diff
@@ -72,18 +79,65 @@ def __init__(
         self.serializer = serializer
         self.kwargs = kwargs
 
-    def get_consumers(self, Consumer: kombu.Consumer, channel: Channel) -> Sequence[kombu.Consumer]:
+    def get_consumers(self, consumer: kombu.Consumer, channel: Channel) -> Sequence[kombu.Consumer]:
```

Comment:
I think we should leave this capitalized, it's like that because that's the signature of this method in `ConsumerMixin`

https://docs.celeryproject.org/projects/kombu/en/stable/reference/kombu.mixins.html

```suggestion
    def get_consumers(self, Consumer: kombu.Consumer, channel: Channel) -> Sequence[kombu.Consumer]:
```